Testing Your Seed.md

```diff
@@ -0,0 +1,34 @@
+# Testing Your Seed
+
+Once you have made the backup of a Seed, the same can be used to access your Skywallet. However, if you have multiple Wallets you will have as many Seeds as well.
+
+In such cases, there is a probability that you can mix up the Seed of a particular Wallet with the Seed of another Wallet.
+
+The **Confirm Seed** button in the Hardware Wallet window of your Desktop Wallet can help you in such situations.
+
+Right after you have clicked on the **Create Backup** button on the Hardware Wallet window and completed the process of the Seed backup, "Create Backup" button will be replaced by the "Confirm Seed" button.
```

Comment:
I do not understand why it goes from Confirm Seed to Create Backup.

Reply:
I was trying to explain that initially this button will not available. Have added an extra sentence to make it more clear.